config/protector.php

```diff
@@ -33,7 +33,7 @@
     | 8: minute
     |
```

Comment:
9: ???

Reply:
and is it really a "parameter order"? cant the user use them in any order they like?